src/sandip/Test_2.java

```diff
@@ -0,0 +1,42 @@
+package sandip;
+
+public class Test_2 {
+
+	void singleOccurance(int[] s2) {
+		for (int index = 0; index < s2.length; index++) {
+			int count = 0;
+			int n1 = s2[index];
+			for (int index1 = 0; index1 < s2.length; index1++) {
+				if (n1 == s2[index1]) {
+					count++;
+				}
+			}
+			if (count == 1) {
+				System.out.print(n1 + " ");
+			}
+		}
+	}
+
+	void pairOfArray(int[] s2) {
+		System.out.println();
+		System.out.println();
+		for (int index = 0; index < s2.length; index++) {
+			int n1 = s2[index];
```

Comment:
you don't require to capture in variable, inside the loop you can directly write